app/controllers/application_controller.rb

```diff
@@ -2,4 +2,13 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+  
+  def user_notifications
+  	if current_user.role=="admin"
+  	Notification.where(to: "admin").all
+  	else
+  	Notification.where(to: current_user.id).all
+  	end
```

Comment:
what if the notification count is greater than 100 (or any large number), won't that be an expensive query? try adding read/unread type statuses to your notification.